### PR TITLE
Extend ruleset with MITRE attack technique ID data

### PR DESCRIPTION
**Pull request by danimegar**
| Branch | Related issue |
|---|---|
| [3.10-mitre](https://github.com/wazuh/wazuh-ruleset/compare/3.11...3.10-mitre) | #500 |

### Description

About 250 high-level rules have been enhanced with [MITRE-ATT&CK](https://attack.mitre.org) information, which are related to brute-force attacks, multiples authentication or password attempts,  account creations or deletions, addition of privilegies to a user, etc.

### Configuration and alert
An example of setting a rule with MITRE and its corresponding alert are shown below.
```
<rule id="5712" level="10" frequency="8" timeframe="120" ignore="60">
    <if_matched_sid>5710</if_matched_sid>
    <description>sshd: brute force trying to get access to </description>
    <description>the system.</description>
    <mitre>
      <id>T1110</id>
    </mitre>
    <same_source_ip />
    <group>authentication_failures,pci_dss_11.4,pci_dss_10.2.4,pci_dss_10.2.5,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_SI.4,nist_800_53_AU.14,nist_800_53_AC.7,</group>
  </rule> 
```
```
{
	"timestamp": "2019-11-08T12:05:26.546+0100",
	"rule": {
		"level": 10,
		"description": "sshd: brute force trying to get access to the system.",
		"id": "5712",
		"mitre": {
			"id": [
				"T1110",
				"T1021"
			],
			"tactics": [
				"Credential Access",
				"Lateral Movement"
			]
		},
		"frequency": 8,
		"firedtimes": 1,
		"mail": false,
		"groups": [
			"syslog",
			"sshd",
			"authentication_failures"
		],
		"pci_dss": [
			"11.4",
			"10.2.4",
			"10.2.5"
		],
		"gdpr": [
			"IV_35.7.d",
			"IV_32.2"
		],
		"hipaa": [
			"164.312.b"
		],
		"nist_800_53": [
			"SI.4",
			"AU.14",
			"AC.7"
		]
	},
	"agent": {
		"id": "001",
		"name": "centos-agent1"
	},
	"manager": {
		"name": "manager1"
	},
	"id": "1573211126.343098",
	"previous_output": "Jan 24 17:15:47 linux-agent sshd[1635]: Invalid user blimey from 208.103.56.41 port 34372\nJan 24 17:15:47 linux-agent sshd[1635]: Invalid user blimey from 208.103.56.41 port 34372\nJan 24 17:15:47 linux-agent sshd[1635]: Invalid user blimey from 208.103.56.41 port 34372\nJan 24 17:15:47 linux-agent sshd[1635]: Invalid user blimey from 208.103.56.41 port 34372\nJan 24 17:15:47 linux-agent sshd[1635]: Invalid user blimey from 208.103.56.41 port 34372\nJan 24 17:15:47 linux-agent sshd[1635]: Invalid user blimey from 208.103.56.41 port 34372\nJan 24 17:15:47 linux-agent sshd[1635]: Invalid user blimey from 208.103.56.41 port 34372",
	"full_log": "Jan 24 17:15:47 linux-agent sshd[1635]: Invalid user blimey from 208.103.56.41 port 34372",
	"predecoder": {
		"program_name": "sshd",
		"timestamp": "Jan 24 17:15:47",
		"hostname": "linux-agent"
	},
	"decoder": {
		"parent": "sshd",
		"name": "sshd"
	},
	"data": {
		"srcip": "208.103.56.41",
		"srcport": "34372",
		"srcuser": "blimey"
	},
	"location": "/var/log/auth.log"
}
```

### CODE REVIEW

**Comment by danimegar** [comment]
Currently, MITRE information have been added to 321 rules, the majority being between level 7 and 15.

**Comment by danimegar** [comment]
Now, MITRE information has been added to 490 rules, the majority being between level 7 and 15.

File: rules/0195-named_rules.xml [hunk 1]
```diff
@@ -329,6 +335,10 @@
     <if_matched_sid>12108</if_matched_sid>
     <same_source_ip />
     <description> Multiple query (cache) failures.</description>
+    <mitre>
+      <id>T1518</id>
```
Comment (chemamartinez) [inline rules/0195-named_rules.xml]: This technique talks about software discovery (installed on the system). This rule is about DNS queries.

File: rules/0485-cylance_rules.xml [hunk 2]
```diff
@@ -93,7 +102,10 @@
     <rule id="87053" level="4">
     	<if_sid>87050</if_sid>
     	<field name="cylance_threats.file_status">abnormal</field>
-    	<description>Cylance Threat: File $(cylance_threats.file_path) is abnormal</description>
+        <description>Cylance Threat: File $(cylance_threats.file_path) is abnormal</description>
```
Comment (vikman90) [inline rules/0485-cylance_rules.xml:105]: Please review indentation.

Comment (danimegar) [reply rules/0485-cylance_rules.xml]: Fixed.

**Pull request merged**